test/integration/test_secrets.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+TEST_ACCOUNT="osd-staging-1"
```

Comment:
`TEST_AWS_ACCOUNT` just to make it more clear? Trying to be explicit to avoid overlap between AWS Account and Account CR

Reply:
Oh, if this is the Acount CR name, then can we change the name of it. And the var

Reply:
@iamkirkbater can we please rename this `TEST_ACCOUNT` CR Name? Having the same name as the Payer account can be misleading.